dotnet8/web/README.md

```diff
@@ -0,0 +1,42 @@
+# Cookiecutter SAM for dotNet based Lambda functions
+
+This is a [Cookiecutter](https://github.com/audreyr/cookiecutter) template to create a Serverless Hello World App based on Serverless Application Model (SAM) and dotnet 3.1.
```

Comment:
I see it's also 3.1 in the dotnet6 one. It's easy to miss. I wonder if we can omit the dotnet version number at all.

```suggestion
This is a [Cookiecutter](https://github.com/audreyr/cookiecutter) template to create a Serverless Hello World App based on Serverless Application Model (SAM) and dotnet.
```

Reply:
Updated both. Good catch